spec/src/main/asciidoc/required-apis.asciidoc

```diff
@@ -41,34 +37,58 @@ MicroProfile applications that make use of APIs or versions not mentioned below
  - <<mp-opentracing, MicroProfile OpenTracing 2.0>>
  - <<mp-rest-client, MicroProfile Rest Client 2.0>>
 
+A MicroProfile compatible implementation must pass *all* the TCKs provided by the MicroProfile individual
+specifications. The Jakarta specifications TCKs are optional.
+
+These APIs and versions guarantee application developers a minimum set of APIs and are not intended to be restrictive.
+
 [[javase]]
-=== Java SE 8
+=== Java SE
 
 Libraries in use by the MicroProfile leverage new features provided in Java SE 8, therefore Java SE 7 and before are unsupported.
-MicroProfile 4.x implementations may provide support for Java SE 9 and beyond, however Java SE 8 compatibility must be maintained.
+MicroProfile implementations are free to provide support for any Java SE 8 and beyond.
 
 [[jakarta-cdi]]
 === Jakarta Contexts and Dependency Injection (CDI) 2.0
 
-Jakarta CDI provides the base for a growing number of APIs included in MicroProfile 4.0.
+Jakarta CDI provides the base for a growing number of APIs included in MicroProfile.
 Use of implementations beyond CDI 2.0 in MicroProfile is allowed, however, not required.
 
  - https://jakarta.ee/specifications/cdi/2.0/cdi-spec-2.0.html
  - http://cdi-spec.org/
 
+==== Optional Requirements
+
+The following CDI features are optional:
+
+ - @ConversationScoped
+ - Expression Language
+ - EJB Integration
+ - Portable Extensions
+
 [[jakarta-jaxrs]]
 === Jakarta RESTful Web Services (JAX-RS) 2.1
 
-Jakarta JAX-RS provides both standard client and server APIs for RESTful communication by MicroProfile 4.0 applications.
-Use of implementations beyond JAX-RS 2.1 in MicroProfile is allowed, however, not required as of version 4.0 of MicroProfile.
+Jakarta JAX-RS provides both standard client and server APIs for RESTful communication by MicroProfile applications.
+Use of implementations beyond JAX-RS 2.1 in MicroProfile is allowed, however, not required.
 
  - https://jakarta.ee/specifications/restful-ws/2.1/restful-ws-spec-2.1.html
  - https://github.com/eclipse-ee4j/jaxrs-api
 
+==== Optional Requirements
+
+The following JAX-RS features are optional:
```

Comment:
Since we don't have the complete list on what are optional, I suggest instead of highlighting the list, we should just say "Some of JAX-RS features are not required, such as Servlet integration, Managed Beans Integration, EJB Integration, etc.". Please refer to the individual implmentation on their support statement.